Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.16.0
+FROM node:18.16.0-alpine
```

Comment:
FROM node:18.20-alpine

I suggest bumping to this, since it still seems to work ok and has ~330 security warnings when you run it through `grype`